src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-pdfReportArtifact.html

```diff
@@ -1,6 +1,7 @@
 <div>
 <br>
-Generate a test report in PDF format, <b>testreport.pdf</b>, and save it in the <b>matlabTestArtifacts</b> folder of the Jenkins workspace.
+Generate a test report in PDF format. By default, MATLAB names the artifact <b>testreport.pdf</b> and stores it in the <b>matlabTestArtifacts</b> folder of the project workspace. 
+To override the default artifact name and location, specify a path relative to the project folder in the <b>File path</b> box. If the text box is empty, MATLAB does not generate an artifact.
```

Comment:
In the note:
<b>Note:</b>This feature is not currently supported on MacOS platforms.

1) A white space is missing between Note: and This.
2) We can say macOS or Mac, but not MacOS.

Reply:
Done.